src/main/java/seedu/cafectrl/data/dish/Ingredient.java

```diff
@@ -38,4 +38,8 @@ public String toString() {
         return name + " - " + qty + unit;
     }
 
+    @Override
+    public boolean equals(Object obj) {
```

Comment:
May I clarify where this new Overriden function of 'equals' is used? It does not seem to be used in any of the changes you made in this PR either unless I missed it while looking through :) Perhaps you may want to take it out if it is not used.

Reply:
it is used when i call assertEquals to compare two lists of ingredients! Java will use this overridden method to compare the ingredient object instead of the original equals method from the Object class which only compares the reference of the object.

Reply:
Okay, thank you for pointing that out!